classes/favs.rb

```diff
@@ -0,0 +1,9 @@
+class Favs
+  attr_accessor :zombie
+
+  # Run this block on startup
+  def initialize
+  end
```

Comment:
We weren't sure if we were going to need it or not, and it got overlooked in the code cleanup. Will remove.